test/core/end2end/h2_ssl_cert_test.cc

```diff
@@ -370,6 +383,12 @@ int main(int argc, char** argv) {
   grpc_init();
   ::testing::InitGoogleTest(&argc, argv);
   int ret = RUN_ALL_TESTS();
+#ifndef OPENSSL_IS_BORINGSSL
```

Comment:
Can you make a separate test instead of rerunning `RUN_ALL_TESTS` with the different configuration? `RUN_ALL_TESTS` is expected to be called at once. ([ref](https://github.com/google/googletest/blob/master/googletest/docs/primer.md))

Reply:
This single configuration re-runs all the tests (20 of them with a new configuration). Reusing the same tests is this feature is about. It also future proofs any new tests that gets added to the this file.

Reply:
Moved the tests into their own section and removed calling RUN_ALL_TESTS multiple times.